test/e2e-setup/bicep/cluster-only.bicep

```diff
@@ -4,6 +4,9 @@ param persistTagValue bool = false
 @description('Name of the hypershift cluster')
 param clusterName string
 
+@description('ControlPlane OpenShift Version ID')
+param openshiftControlPlaneVersionId string = '4.19'
```

Comment:
what's the reason for keeping this to 4.19? Is it because we haven't changed the default version yet? 🤔 same with the node pool version below https://github.com/Azure/ARO-HCP/blob/296c62e356645c7670338d0f8b10c0cada813450/test/e2e-setup/bicep/demo.bicep#L11